kpis/kpi_calculator.py

```diff
@@ -532,6 +532,44 @@ def get_computational_time_ratio(self, plot=False):
 
         return time_rat
 
+    def _set_last_index(self,label):
+        '''Set last index for kpi calcualtion.
+
+        Parameters
+        ----------
+        label: str
+            Name of point to get data for from case.y_store
+
+        '''
+
+        # Initialize index
+        if self.case.y_store['time'].size > 0:
+            i = len(self.case.y_store['time'])-1
+        else:
+            i = 0
+        setattr(self, 'i_last_{}'.format(label),i)
+
+    def _get_data_from_last_index(self,key,t):
+        '''Get data from last index indicated by t.
+
+        Parameters
+        ----------
+        key: str
+            Name of point to get data for from case.y_store
+        t: num
```

Comment:
Again, this description was not updated from an earlier version of the function.  I'm sorry and thanks for catching.  ``t`` here should actually be an integer, since it represents an index.  As such, I suggest changing it to ``i`` and updating the description accordingly in https://github.com/ibpsa/project1-boptest/pull/290/commits/7d63d36db8b5fd8ba53d7345f426827197638b96.

Reply:
Looking more clear now, thanks!